src/tracks/playabletrack/wavetrack/ui/SpectrumVZoomHandle.cpp

```diff
@@ -133,8 +133,8 @@ void SpectrumVZoomHandle::DoZoom(
    const bool spectrumLinear =
       (SpectrogramSettings::Get(*pTrack).scaleType == SpectrogramSettings::stLinear);
 
-
-   bool bDragZoom = WaveChannelVZoomHandle::IsDragZooming(zoomStart, zoomEnd);
+   bool hasDragZoom = true;
```

Comment:
You might qualify this little variable as `const` or to be even more modern, `constexpr`.  That's a minor bit of style.  I skip it sometimes when the scope of a variable is small, but it can be helpful if ever you have something at the top of a large function or block.